.travis.yml

```diff
@@ -1,2 +1,8 @@
-language: c
-script: bin/shpec
+language: sh
+script:
+  - time bash bin/shpec -f 1 shpec/shpec_shpec.sh
+  - time dash bin/shpec -f 1 shpec/shpec_shpec.sh
+  - time ksh  bin/shpec -f 1 shpec/shpec_shpec.sh
```

Comment:
@locochris  indeed it's the `end` keyword. :)

Question is *where the good place to disable it?*.
So far I did it in a wrapping alias. (to turn it on and off). But I thinks it might better to get this inside the script itself.